lib/matplotlib/patches.pyi

```diff
@@ -74,7 +74,7 @@ class Patch(artist.Artist):
 
 class Shadow(Patch):
     patch: Patch
-    def __init__(self, patch: Patch, ox: float, oy: float, **kwargs) -> None: ...
+    def __init__(self, patch: Patch, ox: float, oy: float, *, shade: float = …, **kwargs) -> None: ...
```

Comment:
```suggestion
    def __init__(self, patch: Patch, ox: float, oy: float, *, shade: float = ..., **kwargs) -> None: ...
```

Sorry I think my tablet was trying to be clever on the last one.

Reply:
I thought those ... looked too close together.....